src/main.js

```diff
@@ -6,23 +6,26 @@ import App from './App'
 import router from './router'
 import Vuetify from 'vuetify'
 import 'vuetify/dist/vuetify.min.css'
-import { firebaseApp } from '@/firebase'
+import {firebaseApp} from './firebase'
 
-Vue.config.productionTip = false
+export const eventBus = new Vue()
```

Comment:
Flera Vue instanser alltså. Coolt :)

Reply:
Eller hur! Max får ta cred för den:
https://www.udemy.com/vuejs-2-the-complete-guide/learn/v4/t/lecture/5950984?start=0